integration_test/test_integration.py

```diff
@@ -88,10 +96,16 @@ def test_integration(tmp_path, capfd, ebuild):
     verify_files([(dist_file, pkg_info.checksum)])
     with tarfile.open(dist_file, "r") as tarf:
         for directory in pkg_info.directories:
-            for filename in ("Cargo.lock", "Cargo.toml"):
-                member = tarf.getmember(f"{directory}/{filename}")
+            member = tarf.getmember(f"{directory}/Cargo.toml")
+            tarf.extract(member, tmp_path, set_attrs=False)
+            for current in (Path(directory) / "Cargo.lock").parents:
+                try:
+                    member = tarf.getmember(f"{current}/Cargo.lock")
+                except KeyError:
+                    continue
```

Comment:
Technically speaking, you're not checking if any file was extracted but since we can reasonably assume that some file will be there, it's no big deal.

Reply:
oh, I see. My fault, I have changed it and it sould check if any file was extracted now.